nodes/nodes/eco-node-maintenance-operator.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+[id="node-maintenance-operator"]
+= Using the Node Maintenance Operator to place nodes in maintenance mode
+include::modules/common-attributes.adoc[]
+include::modules/virt-document-attributes.adoc[]
+:context: node-maintenance-operator
+toc::[]
+
+You can use the Node Maintenance Operator to place nodes under maintenance mode. This is a standalone version of the Node Maintenance Operator that is independent of OpenShift Virtualization installation.
```

Comment:
You can use the Node Maintenance Operator to place nodes under maintenance mode

maybe rephrase to 

You can use the Node Maintenance Operator to place nodes in maintenance mode.

or 

You can use the Node Maintenance Operator to place nodes under maintenance.